docs/guides/events-api.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: default
+title: "Events API"
+lang: en
+---
+
+# Events API
+
+The [Events API](https://api.slack.com/events-api) used in the following example is a streamlined, easy way to build apps and bots that respond to activities in Slack. All you need is a Slack app and a secure place for us to send your events.
+
+All you need to do to handle Events API requests are:
+
+1. [Verify requests](https://api.slack.com/docs/verifying-requests-from-slack) from Slack
+1. Parse the request body and check if the `type` in `event` is the one you'd like to handle
+1. Whatever you want to do with the event data
+1. Respond with 200 OK reply as aknowledgement
+
+
+Your app has to respond to the request within 3 seconds by `ack()` method for sure. Otherwise, the user will see the timeout error on Slack.
+
+## Bolt Examples
+
+**NOTE**: If you're a beginner to using Bolt for Slack App development, consult [The Basics of Bolt]({{ site.url | append: site.baseurl }}/guides/Bolt), first.
+
+Bolt does most of the things for you. The steps you need to handle would be:
+
+* Specify the `event.type` (and also `event.subtype` [when necessary](https://api.slack.com/events/message#message_subtypes)) to handle
+* Whatever you want to do with the event data
+* Respond with 200 OK as the acknowledgment
+
+In event payloads, `response_url` is not included as it's not a payload coming from direct user interactions. Also, it's not possible to post a message using `ctx.ack()` for the same reason. If an event you receive is a user interaction and you'd like to post a reply to the user at the conversation the event happened, call **chat.postMessage** API or other similar ones with `channel` in the event payload.
```

Comment:
call **chat.postMessage** API -> the [**`chat.postMessage`**](https://api.slack.com/methods/chat.postMessage) method